### PR TITLE
📖 Add KubeCon Detroit 2022 videos to CAPI book

### PR DESCRIPTION
**Pull request by oscr**
**What this PR does / why we need it**:

Adds relevant videos (that I found!) from KubeCon to the CAPI book. 

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: docs/book/src/developer/guide.md [hunk 1]
```diff
@@ -147,6 +148,10 @@ To test another iteration, you'll need to follow the steps to build, push, updat
 
 **Additional ClusterAPI KubeCon talks**
 
+* [How Adobe Planned For Scale With Argo CD, Cluster API, And VCluster - October 2022](https://www.youtube.com/watch?v=p8BluR5WT5w)
+* [Bare-Metal Chronicles: Intertwinement Of Tinkerbell, Cluster API And GitOps - October 2022](https://www.youtube.com/watch?v=NCFUUjTw6hA)
+* [Running Isolated VirtualClusters With Kata & Cluster API - October 2022](https://www.youtube.com/watch?v=T6w3YrExorY)
+* [SIG Cluster Lifecycle Intro - October 2022](https://www.youtube.com/watch?v=0Zo0cWYU0fM)
```
Comment (fabriziopandini) [inline docs/book/src/developer/guide.md:154]: @sbueringer @richardcase what about adding also tutorials + links to tutorials web sites?

**Review by fabriziopandini — COMMENTED** [review]
Thanks for this PR, this is great!

File: docs/book/src/developer/guide.md (hunk 1 above)
Comment (sbueringer) [reply docs/book/src/developer/guide.md:154]: Sounds good

In our case I would link:
* sched with video: https://kccncna2022.sched.com/event/1BZDs (also gives some context in the talk description)
* tutorial repo: https://github.com/ykakarap/kubecon-na-22-capi-lab

Comment (ykakarap) [reply docs/book/src/developer/guide.md:154]: The sched link also has a link to the youtube video so that is a plus.

Comment (sbueringer) [reply docs/book/src/developer/guide.md:154]: Yup that's why I wrote "sched with video" :D

Comment (oscr) [reply docs/book/src/developer/guide.md:154]: Added

File: docs/book/src/developer/guide.md [hunk 2]
```diff
@@ -147,9 +148,18 @@ To test another iteration, you'll need to follow the steps to build, push, updat
...
 * [How to Migrate 700 Kubernetes Clusters to Cluster API with Zero Downtime - May 2022](https://www.youtube.com/watch?v=KzYV-fJ_wH0)
 * [Build Your Own Cluster API Provider the Easy Way - May 2022](https://www.youtube.com/watch?v=HSdgmcAAXa8)
 
+**Tutorials**
+
+* - [kubectl Create Cluster: Production-ready Kubernetes with Cluster API 1.0 - October 2022](https://kccncna2022.sched.com/event/1BZDs)
```
Comment (sbueringer) [inline docs/book/src/developer/guide.md:160]: Formatting `* -` looks slightly off

File: docs/book/src/developer/guide.md (hunk 1 above)
Comment (sbueringer) [reply docs/book/src/developer/guide.md:154]: Let's wait a bit what Richard says, he had another separate tutorial (~ building a provider)

Comment (richardcase) [reply docs/book/src/developer/guide.md:154]: Sorry for the delay, for our tutorial:

- https://kccncna2022.sched.com/event/182Ha/tutorial-so-you-want-to-develop-a-cluster-api-provider-anusha-hegde-nirmata-winnie-kwon-vmware-richard-case-suse-avishay-traeger-red-hat
- https://capi-samples.github.io/kubecon-na-2022-tutorial/

File: docs/book/src/developer/guide.md (hunk 2 above)
Comment (oscr) [reply docs/book/src/developer/guide.md:160]: It's correctly formatted if that's what you mean. I created a sublist to make it clear it's video + lab link.

File: docs/book/src/developer/guide.md (hunk 1 above)
Comment (oscr) [reply docs/book/src/developer/guide.md:154]: Thank you Richard! Added.

**Comment by stmcginnis** [comment]
/lgtm

Great to have these resources added to the site!

File: docs/book/src/developer/guide.md (hunk 2 above)
Comment (sbueringer) [reply docs/book/src/developer/guide.md]: That looks a bit strange to me :)

image

What about just indenting the `source code` items?

Comment (sbueringer) [reply docs/book/src/developer/guide.md]: (or adding it in `()` to the items with the sched links)

Comment (oscr) [reply docs/book/src/developer/guide.md]: No problem. That works for me :)

**Review by killianmuldoon — COMMENTED** [review]
Thanks for this! 

/lgtm

**Comment by sbueringer** [comment]
Thank you very much!

/lgtm
/approve

/cherry-pick release-1.3

**Pull request merged**